pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -70,5 +75,135 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			Expect(err).To(HaveOccurred())
 
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// sa can create project
+		ginkgo.It("sa can create project", func() {
+
+			h.SetServiceAccount("system:serviceaccount:%s:dedicated-admin-project")
+
+			_, err := createNamespace("cust-ns", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// regular dedicated-admin user can create admin rolebinding
+		ginkgo.It("can create admin rolebinding", func() {
+
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
+			daGroup, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			h.Impersonate(rest.ImpersonationConfig{
+				UserName: "dummy-admin@redhat.com",
+				Groups: []string{
+					"dedicated-admins",
+				},
+			})
+			defer func() {
+				h.Impersonate(rest.ImpersonationConfig{})
+			}()
+
+			_, err = createRolebinding("dummy-user@redhat.com", "admin", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// can create edit rolebinding
+		ginkgo.It("can create edit rolebinding", func() {
+
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
```

Comment:
didn't we add that user to dedicated-admins in the previous test? I'd export that into a parent ginko.context/It(). I didn't get why we're having that user as dedicated admin though the addition is being done by dummy-admin@redhat.com(from h.impersonate)

Reply:
+1 I will create another PR to refactor these tests